test/functional/api/autocmd_spec.lua

```diff
@@ -881,6 +881,93 @@ describe('autocmd api', function()
         eq([[:echo "Buffer"]], normalized_aus[1].command)
       end)
     end)
+
+    describe('id', function()
+      local _id, _id1
+      after_each(function()
+        api.nvim_del_autocmd(_id)
+        if _id1 then
+          api.nvim_del_autocmd(_id1)
+          _id1 = nil
+        end
+      end)
+
+      it('retrieves events using an ID', function()
```

Comment:
"retrieves" is an overly verbose word
```suggestion
      it('gets events by ID', function()
```